src/plugins/ecs/__init__.py

```diff
@@ -479,5 +527,81 @@ def tasks_get_lookup_term(text):
                 return data[(data.lower().find('---') + 3):]
 
 
-
-
+#retrieve json data from super jenkins for build urls
+def get_superjenkins_data(superjenkins_link,beginning_script_tag,ending_script_tag):
+
+    # get json to get individual services/application build links
+    cached_items = ""
+    cached_array = ""
+
+    try:
+        returned_data = requests.get(superjenkins_link)
+        returned_data_iterator = returned_data.iter_lines()
+
+        for items in returned_data_iterator:
+            if beginning_script_tag in items:
+                cached_items = items.replace(beginning_script_tag, "").replace(ending_script_tag, "")
+                break
+
+        for items in json.loads(cached_items):
+            cached_array = json.loads(cached_items)[items]
+
+    except Exception, e:
+        print "Error in retrieving and creating json for build urls ==> " + str(e)
+
+    return cached_array
+
+
+#retrieve data from config files for compare
+def get_in_ecs_compare_data(config, args, args_eval):
+
+	result = dict()
+
+	if args_eval == 3:
+		result['cluster_name'] = None
+		result['region_name'] = args[1]
+		result['account'] = args[2]
+	elif args_eval == 4:
+		result['cluster_name'] = args[0]
+		result['region_name'] = args[2]
+		result['account'] = args[3]
+
+	for account in config['Accounts']:
+		if account['AccountName'] == result['account']:
+			result['RoleArn'] = account['RoleArn']
+			for the_region in account['Clusters']:
+				if the_region == result['region_name']:
+					if result['cluster_name'] == None:
+						result['cluster_name'] = account['Clusters'][the_region]['cluster_list']
+					result['environment_code_name'] = account['Clusters'][the_region]['environment_code_name']
+					result['service_exclude_list'] = account['Clusters'][the_region]['service_exclude_list']
+					result['team_name'] = account['Clusters'][the_region]['team_name']
+	if result.has_key('environment_code_name') == False:
```

Comment:
I've never used this - I've always just done, "if 'environment_code_name' in result:", but good to know there is an alternative.